### ISSUE TITLE
Unsupported action 'focus_tile'

### ISSUE THREAD

**Issue by sodface** [project-repo/cagebreak#43]
https://github.com/project-repo/cagebreak/blob/bea18c7a6f0f709251d6e7670eed5d3d9ef1b02b/example_scripts/focus_follows_cursor.sh#L21

Seems to have been broken by:
https://github.com/project-repo/cagebreak/commit/f8df676572535e2a78494a8d172ebf93d3b1cc50

**Comment by project-repo** [comment]
Hi sodface

Thanks for the report! This is actually not a bug: The documentation you
are referring to is located in the section `EVENTS` section which
reports on which events cagebreak *sends* over the socket instead of the
commands which can be sent. We're sorry if that commit caused confusion,
it's related to a WIP that isn't read for release yet.

Cheers
project-repo

**Comment by sodface** [comment]
Hi  project-repo, yes I realized that after I posted the issue and then deleted that portion from the first post. The example script `focus_follows_cursor.sh` is still using the removed command though and causes the unsupported action error to be displayed.

**Comment by project-repo** [comment]
Hi

Sorry about that, I was working with the e-mail notification version
of the post which unfortunately doesn't get updated on edits. You are
right, the current version of the script is broken, it seems that we
missed it when reverting the preliminary changes (hence our new
convention to use feature branches). Thanks for pointing this out, this
script should not have been checked in. We will keep this issue open
until it is decided what to do about it (which depends on whether the
feature enabling this will be included in the next release).

Cheers
project-repo

**Comment by project-repo** [comment]
Hi sodface

The script was removed and moved to the `feature/numeric-commands`
branch (where the features needed to run the script will be developed).
The effort of reimplementing the script using the currently available
features is not really worth it, especially b.c. this is the precise
reason we proposed to implement the new features. Thanks a lot for
pointing this bug out!

Cheers
project-repo
